mintpy/utils/readfile.py

```diff
@@ -1050,6 +1050,7 @@ def get_hdf5_dataset(name, obj):
             fname + '.xml',
             fname + '.par',
             os.path.splitext(fname)[0] + '.hdr',
+            fname + '.hdr',
```

Comment:
Given that the one created with `SUFFIX=ADD` is more unique than the one created with `SUFFIX=REPLACE`, shall we put this in front of the `SUFFIX=REPLACE` version above?

Reply:
It would be good to write comments after these two lines for the `SUFFIX=ADD/REPLACE` option, to easily remind ourselves in the future about this change.